### DISCUSSION TITLE
chatsydney报错

### DISCUSSION THREAD

**Discussion by tzcnbo** [viopsa233/AI-testers-Forum#88] [Q&A]
这是什么情况啊，Sydney没给我发一句话，后面turmux显示5次重试后报错了，求解决
![Screenshot_2023-07-13-18-56-56-29_84d3000e3f4017145260f7618db1d683](https://github.com/viopsa233/AI-testers-Forum/assets/105526202/93f08e99-8149-4e86-9eaa-837ce5452359)

**Comment by viopsa233** [comment]
微软更新检测了,等更新

**Reply by codescipio** [reply]
暗黑大大弃坑之后都不知道去哪找更新了

**Reply by jonasreisner** [reply]
woc,不会要亡了吧

**Reply by jonasreisner** [reply]
试了一下SydneyQt那个解决方法走cloudflare，实测可以

**Reply by tzcnbo** [reply]
求指路

**Reply by viopsa233** [reply]
现在修复了

**Reply by jonasreisner** [reply]
cloudflare那个暂时也挂了  可以参考@SoraRoseous的实现

**Comment by juzeon** [comment]
create conversation那个接口还和以前一样。

但Websocket接口现在要带上cookies才能访问。

另外用解锁的IP现在成了必选项，比如用Cloudflare Workers。
